examples/README.md

```diff
@@ -0,0 +1,67 @@
+# Example connectors
+
+The Native SDK for Connectors provides a couple of examples that show a general pattern, recommended by Snowflake, of 
+creating new connectors as a [Native Applications][Native Apps docs]. 
+From these examples, you will learn:
+* how to create pull-based connectors ([learn more about pull-based pattern][Pull-based pattern])
+* how to create push-based connectors ([learn more about push-based pattern][Push-based pattern])
+* how to implement the connector configuration mechanism
+* how to implement the ingestion mechanism
+* how to create an instance of the application in the Snowflake environment
+* how to use the [SnowSQL][SnowSQL guide] tool in connector development process
+
+## Prerequisites
+
+- Basic knowledge of [Snowflake Native Apps][Native Apps docs]
+- Basic knowledge of SQL/SnowflakeScripting
+- Snowflake user with `accountadmin` role
+
+## Prepare your local environment
+
+Before approaching example connectors, please take up the following steps or make sure that they are currently done:
+- Install Java 11
+- Install [SnowSQL][SnowSQL guide]
+- Configure [SnowSQL][SnowSQL guide] to allow using [variables][SnowSQL variables] (`variable_substitution = True`)
+- Configure [SnowSQL][SnowSQL guide] to [exit on first error][SnowSQL exit on error] (`exit_on_error = True`)
```

Comment:
doesn't it look better?: 

- Configure [SnowSQL][SnowSQL guide] 
    - [exit on first error][SnowSQL exit on error] (`exit_on_error = True`)
    - allow using [variables][SnowSQL variables] (`variable_substitution = True`)